com.unity.visualeffectgraph/Editor/Models/VFXGraph.cs

```diff
@@ -159,7 +159,6 @@ static string[] OnWillSaveAssets(string[] paths)
                 if (vfxResource != null)
                 {
                     var graph = vfxResource.GetOrCreateGraph();
-                    graph.OnSaved();
```

Comment:
Note : This minimal change comes from here : https://github.com/Unity-Technologies/Graphics/pull/3918 and prevent to compile twice. It requires a change which landed a while ago in 2020.3.4f1.
I will backport the test independently. 
🟢